packages/app/src/elements/item-view.ts

```diff
@@ -563,7 +674,7 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
                     class="animated padded spacing evenly stretching horizontal layout save-cancel"
                     ?hidden=${!this._editing}
                 >
-                    <pl-button class="primary spacing horizontal layout" @click=${this.save}>
+                    <pl-button class="primary spacing horizontal layout" @click=${() => this.save()}>
```

Comment:
This was sending an object `{ isTrusted: true }`, but it didn't seem necessary or used, and it messed with my new param.